pkg/destroy/aws/aws.go

```diff
@@ -260,17 +264,18 @@ func (o *ClusterUninstaller) RunWithContext(ctx context.Context) ([]string, erro
 
 // findEC2Instances returns the EC2 instances with tags that satisfy the filters.
 //   deleted - the resources that have already been deleted. Any resources specified in this set will be ignored.
-func (o *ClusterUninstaller) findEC2Instances(ctx context.Context, ec2Client *ec2.EC2, deleted sets.String) ([]string, error) {
+func (o *ClusterUninstaller) findEC2Instances(ctx context.Context, ec2Client *ec2.EC2, deleted sets.String) ([]string, []string, error) {
```

Comment:
Add to the description of this function details about what the different return values are, since it is not obvious from the function signature.